pkg/podmounter/mppod/creator.go

```diff
@@ -251,16 +255,44 @@ func (c *Creator) createCacheVolumeSourceForEmptyDir(volumeAttributes map[string
 	return corev1.VolumeSource{EmptyDir: emptyDir}, nil
 }
 
-// createCacheVolumeSourceForEmptyDir creates an `persistentVolumeClaim` volume source to use as local-cache.
-func (c *Creator) createCacheVolumeSourceForPVC(volumeAttributes map[string]string) (corev1.VolumeSource, error) {
-	claimName := volumeAttributes[volumecontext.CachePersistentVolumeClaimName]
-	if claimName == "" {
-		return corev1.VolumeSource{}, fmt.Errorf("%q must be provided with %q cache type", volumecontext.CachePersistentVolumeClaimName, cacheTypePersistentVolumeClaim)
+// createCacheVolumeSourceForEphemeral creates an `ephemeral` volume source to use as local-cache.
```

Comment:
np: Just noticed a typo on line 200 above: `configuragion in mountOptions")` -> `configuration`

Reply:
Fixed with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/496/commits/b5216c7b894ba5b2c52f65a2a9e4e7d519eeb501